src/views/landingpage/landingpage.js

```diff
@@ -72,6 +72,7 @@ class LandingPage extends React.Component {
             <LinkButton text="WHY?" onPress={() => window.alert('WIP')} />
           </View>
           <SwapTab
+            loading={Object.keys(rates).length === 0 && currencies.length === 0}
```

Comment:
I would go for 
```suggestion
            loading={Object.keys(rates).length === 0 || currencies.length === 0}
```

because if one of those two variables isn't loaded correctly nothing should be shown to the user.